io.github.antimicrox.antimicrox.yml

```diff
@@ -0,0 +1,24 @@
+app-id: io.github.antimicrox.antimicrox
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: antimicrox
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
```

Comment:
You already have device=dri with device=all

Reply:
Thanks, removed!